packages/vitest/src/node/pools/browser.ts

```diff
@@ -46,7 +46,16 @@ export function createBrowserPool(ctx: Vitest): ProcessPool {
 
     const promise = waitForTests(project, files)
 
-    await provider.openPage(new URL('/', origin).toString())
+    const orchestrators = project.browserRpc.orchestrators
```

Comment:
Since we now have direct RPC to every orchestrator, we can just ask them to run tests